jsfuck.js

```diff
@@ -105,9 +105,9 @@
     '^':   null,
     '_':   null,
     '`':   null,
-    '{':   '(true+[]["flat"])[20]',
+    '{':   '(undefined+[]["at"])[23]',
```

Comment:
* `([]["at"]+"")[14]` would be 68 characters
* `([0]+false+[]["at"])[20]` would be 63 characters